tests/storages_tests/rdb_tests/test_with_server.py

```diff
@@ -26,13 +29,33 @@ def objective(trial: optuna.Trial) -> float:
     return f(x, y)
 
 
-def run_optimize(args: Tuple[str, str]) -> None:
+def run_optimize(args: Tuple[str, str, int, bool]) -> str:
```

Comment:
I prefer defining another function for the progress bar to extend this `run_optimize`. What do you think?

Reply:
Ok, I defined another function.